monitor_other_boards.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+from datetime import datetime
+import subprocess
+
+import slack_reporter
+
+#                SSH target               name
+other_boards = (("odroid@odroid",         "odroid-C4"),
```

Comment:
Not sure about this name: it's arguably `all_boards` because we don't know which one is the other board. Yes, this means that the monitor is going to SSH into itself.